src/satellite/jest.config.js

```diff
@@ -0,0 +1,19 @@
+// Define the env variables our code expects
```

Comment:
We need a follow-up to wire this project into our Jest config too (e.g., define it in our list of projects).  @menghif same question: can you fix in a follow-up or file?

Reply:
Will file it